src/PioneerUtils.jsm

```diff
@@ -16,7 +17,36 @@ const PIONEER_ID_PREF = "extensions.pioneer.cachedClientID";
 
 // Make crypto available and make jose use it.
 Cu.importGlobalProperties(["crypto"]);
-setCrypto(crypto);
+joseSetCrypto(crypto);
+
+/**
+ * @typedef {Object} PioneerUtilsConfig
+ * @property {String} studyName
+ *   Unique name of the study.
+ *
+ * @property {String} studyVersion
+ *   Version of the study. Should match contents of install.rdf.
```

Comment:
The only thing this version is used for is schema selection, it doesn't need to match install.rdf.

Reply:
In that case, how would you feel about making this `schemaVersion`, while internally translated it into `studyVersion` when we pass things on to Telemetry?

Reply:
wfm